Doc/library/asyncio-platforms.rst

```diff
@@ -61,22 +63,19 @@ hardware (availability of `HPET
 <https://en.wikipedia.org/wiki/High_Precision_Event_Timer>`_) and on the
 Windows configuration.
 
+.. versionchanged:: 3.8
+
+   On Windows, :class:`ProactorEventLoop` is now the default event loop.
+
 
 .. _asyncio-windows-subprocess:
 
 Subprocess Support on Windows
 -----------------------------
 
-:class:`SelectorEventLoop` on Windows does not support subproceses.
-On Windows, :class:`ProactorEventLoop` should be used instead::
-
-  import asyncio
-
-  asyncio.set_event_loop_policy(
-      asyncio.WindowsProactorEventLoopPolicy())
-
-  asyncio.run(your_code())
+The default event loop :class:`ProactorEventLoop` supports subprocesses.
 
+On Windows, :class:`SelectorEventLoop` does not support subprocesses.
```

Comment:
On Windows, the default event loop :class:`ProactorEventLoop` supports subprocesses, whereas :class:`SelectorEventLoop` does not.

Reply:
done